components/homme/src/test_src/dcmip2016-baroclinic.F90

```diff
@@ -182,7 +182,7 @@ SUBROUTINE baroclinic_wave_test(deep,moist,pertt,X,lon,lat,p,z,zcoords,u,v,t,the
     inttau2 = constC * z * exp(- scaledZ**2)
 
     ! radius ratio

```

Comment:
what does 'deep' option do here? iirc i ran both branches with it and did not see anything.